extension/src/popups/MinervaMenu.js

```diff
@@ -1,6 +1,7 @@
-import { renderStudentRankings } from "./MinervaMenuRenderInfo.js";
+import { renderCurrentStudentInfo, renderStudentRankings } from "./MinervaMenuRenderInfo.js";
 
 function transitionToMainMenu(studentName) {
+    diplayCurrentStudentInfo(studentName);
```

Comment:
See my comments inside `ApiInterface`. I want to move away from explicitly doing local storage get, instead using `GetStudentName` message eventually.

Reply:
Moved to getStudentName